Update turn-off-services.md (#2366)
RedisLabs/redislabs-docs: content/rs/clusters/optimize/turn-off-services.md

- [LDAP authentication]({{< relref "/rs/security/ldap/_index.md" >}}) - `saslauthd`
- [Discovery service]({{< relref "rs/networking/cluster-dns.md" >}})- `mdns_server`, `pdns_server`
- [Active-Active databases]({{< relref "/rs/databases/active-active/_index.md" >}}) - `crdb_coordinator`, `crdb_worker`

> - Alert Manager - `alert_mgr` (We recomend you to disable this service only if you have alterative alert system)

Reviewer: ```suggestion
- Alert Manager - `alert_mgr` (For best results, disable only if you have an alternate alert system.)
```
